cmd/fleetctl/get.go

```diff
@@ -1021,6 +1075,18 @@ func getUserRolesCommand() *cli.Command {
 	}
 }
 
+func printQueryTable(c *cli.Context, columns []string, data [][]string, nInheritedQueries *int) {
+	table := defaultTable(c.App.Writer)
+	table.SetHeader(columns)
+	table.SetReflowDuringAutoWrap(false)
+	table.AppendBulk(data)
+	table.Render()
+
+	if nInheritedQueries != nil {
+		fmt.Printf("Not showing %d inherited queries. To see global queries, run this command without the `--team` flag.\n", *nInheritedQueries)
```

Comment:
Is this function only called with the team option?
If so should we not rename it to `printQueryTableForTeams` ?

Reply:
No it is called also when listing global queries.